src/WheelBase.cpp

```diff
@@ -290,12 +290,14 @@ bool WheelBase::Select()	// return true if this selection can end the screen
 			RString sThisItemSectionName = m_CurWheelItemData[m_iSelection]->m_sText;
 			if( m_sExpandedSectionName == sThisItemSectionName ) // already expanded
 			{
+				GAMESTATE->sLastOpenSection = sThisItemSectionName; // remember this section
```

Comment:
Can put this outside of the if/else so it's not duplicated. Would also change the comment to something like:

```cpp
// Keep track of the open section so that we can restore it
// when navigating back to ScreenSelectMusic.
GAMESTATE->sLastOpenSection = sThisItemSectionName;
```

By itself, the current comment adds nothing of value (why are we remembering this section? etc.).